enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemBanner.tsx

```diff
@@ -61,88 +51,53 @@ export const SdkUsageProblemBanner = ({
       <Popover.Target>
         <Flex
           onClick={() => setExpanded(!expanded)}
-          className={cx(S.Indicator, isError ? S.Error : S.Warning)}
+          className={cx(S.Indicator)}
```

Comment:
We no longer differentiate the colors from the severity per the design. We always show the same blue color regardless of whether it's an error or a warning.